.github/workflows/release-upload.yaml

```diff
@@ -25,20 +25,21 @@ jobs:
     name: "Wait for hydra check-runs"
     runs-on: ubuntu-latest
     steps:
-    - name: Get specific check run status
+    - name: Waiting for ci/hydra-build:required to complete
```

Comment:
Updated to the same as in `release-ghcr.yaml`. Seems it was copied from an example and then forgotten to be updated.